app/uk/gov/hmrc/cipphonenumberverification/connectors/CircuitBreakers.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cipphonenumberverification.connectors
+
+import akka.pattern.CircuitBreaker
+import akka.stream.Materializer
+import play.api.Logging
+import uk.gov.hmrc.cipphonenumberverification.config.CircuitBreakerConfig
+
+import scala.concurrent.Future
+import scala.concurrent.duration.Duration
+import scala.util.Try
+
+trait CircuitBreakers extends Logging {
```

Comment:
should this be singular?

Reply:
No its a trait

Reply:
No, I mean the name, singular, as in CircuitBreaker instead of CircuitBreakers (plural)